apis/v1alpha1/types.go

```diff
@@ -32,12 +32,6 @@ type ProviderConfigSpec struct {
 	// automatically inject Terraform provider configuration blocks.
 	// +optional
 	Configuration *string `json:"configuration,omitempty"`
-
-	// PluginCache enables terraform provider plugin caching mechanism
-	// https://developer.hashicorp.com/terraform/cli/config/config-file#provider-plugin-cache
-	// +optional
-	// +kubebuilder:default=true
-	PluginCache *bool `json:"pluginCache,omitempty"`
```

Comment:
Can we test upgrading a running version of provider-terraform to this version to make sure this doesn't break? It's v1alpha1 but may be worth bumping to v1alpha2 and announce it as breaking change. I'd expect it to work as expected since its `omitempty` already but a validation would be good.